apps/wing-console/console/ui/src/features/logs-pane/console-logs.tsx

```diff
@@ -129,7 +157,7 @@ const LogEntryRow = memo(
                 "flex-shrink-0 select-none pointer-events-none",
               )}
             >
-              {dateTimeFormat.format(log.timestamp)}
+              {formatDate(log.timestamp)}
```

Comment:
Everytime there's a new log entry, all previou log entries will be formatted again, right?

Why don't we add a `<LogEntry />` component and use a memo so we format the date only once?